<launch>

  <!-- <node name="auto_dock" pkg="rr_auto_dock" type="auto_dock.py" >
  </node> -->

  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen">
    <param name="video_device" value="/dev/video0" />
    <!-- <param name="camera_frame_id" value="front_camera_link" /> -->
    <param name="camera_frame_id" value="camera" />
    <param name="image_width" value="1280" />
    <param name="image_height" value="720" />
    <param name="framerate" value="30" />
    <param name="pixel_format" value="yuyv" />
    <!-- <param name="camera_info_url" value="package://rr_auto_dock/misc/ost.yaml"/>not sure if this is working -->
    <remap from="/usb_cam/image_raw" to="/camera"/>
    <remap from="/usb_cam/camera_info" to="/camera_info"/>
  </node> 

  <!-- <param name="robot_description" textfile="$(find twgc_payload)/rvizconfig/urdf/twgc.urdf" /> -->
  <include file="$(find mycar_description)/launch/car.launch" />

  <!-- <node pkg="robot_state_publisher" type="robot_state_publisher" name="robot_state_publisher">
    <param name="use_tf_static" value="true"/>
  </node> -->

  <node pkg="aruco_detect" name="aruco_detect"
    type="aruco_detect" output="screen" respawn="false">
    <param name="publish_images" value="false" />
    <param name="fiducial_len" value="0.13"/>
    <param name="dictionary" value="7"/>
    <param name="do_pose_estimation" value="true"/>
  </node>

</launch>
